CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 0.10.x
 
 Breaking changes:
+- [#1535](https://github.com/rails-api/active_model_serializers/pull/1535) Move the adapter and adapter folder to
```

Comment:
I mean, like ArraySerializer -> CollectionSerializer per the RFC https://github.com/rails-api/active_model_serializers/blob/master/docs/rfcs/0000-namespace.md#keeping-compatibility